rsconnect/actions.py

```diff
@@ -650,9 +677,9 @@ def gather_basic_deployment_info_for_notebook(connect_server, app_store, file_na
             raise api.RSConnectException('Cannot change app mode to "static" once deployed. '
                                          'Use --new to create a new deployment.')
 
-    title = title or _default_title(file_name)
+    title, default_title = (title, False) if title else (_default_title(file_name), True)
```

Comment:
This took me a couple of readings to grok. Perhaps this would read better?
```suggestion
    default_title = not bool(title)
    title = title or _default_title(file_name)
```

Reply:
Yeah, I like that much better.  It was the `bool(...)` bit of magic I wasn't sure how to do.  Thanks for the reminder.